src/modules/numberplate/bikeProvinceCodes.skip.ts

```diff
@@ -0,0 +1,158 @@
+import { createProvinceHashTable } from "./helpers";
+import { ProvinceDataset } from "./types.skip";
+
+export const bikeProvinceDataset: ProvinceDataset = [
```

Comment:
I think `Bike` is not that word what we want and `Motorcycle` is the right word instead.